src/xenia/kernel/kernel_state.cc

```diff
@@ -326,16 +326,22 @@ void KernelState::SetExecutableModule(object_ref<UserModule> module) {
           // As we run guest callbacks the debugger must be able to suspend us.
           dispatch_thread_->set_can_debugger_suspend(true);
 
+          auto global_lock = global_critical_region_.Acquire();
+          global_lock.unlock();
```

Comment:
AcquireDeferred -> std::defer_lock_t?

Reply:
Are you suggesting to disconnect this code from the GCR?

Reply:
No, std::unique_lock constructor with std::defer_lock_t can create a lock object without actually locking.

Reply:
Ah, I wasn't aware. I'll take a look, thanks.